### PR TITLE
Fix for vic machine server publish

### PR DESCRIPTION
**Pull request by mdharamadas1**
Fix for vic machine server image tagging for publish release.

### CODE REVIEW

**Comment by anchal-agrawal** [comment]
Looks like there's an issue with the yaml file.

File: .drone.yml [hunk 1]
```diff
@@ -258,6 +258,8 @@ pipeline:
     image: plugins/gcr
     repo: eminent-nation-87317/vic-machine-server
     dockerfile: cmd/vic-machine-server/Dockerfile
+    environment:
+      - DRONE_TAG:  ${DRONE_TAG}
```
Comment (anchal-agrawal) [inline .drone.yml]: Looks like the correct format for specifying env variables doesn't use the `-` character (see the `vic-integration-test-on-pr` step).

Comment (mdharamadas1) [reply .drone.yml]: ah thanks for pointing that out, fixed!

File: .drone.yml [hunk 2]
```diff
@@ -258,6 +258,8 @@ pipeline:
     image: plugins/gcr
     repo: eminent-nation-87317/vic-machine-server
     dockerfile: cmd/vic-machine-server/Dockerfile
+    environment:git
```
Comment (anchal-agrawal) [inline .drone.yml]: Stray `git` here?

Comment (mdharamadas1) [reply .drone.yml]: crap, sorry fixed !

**Comment by anchal-agrawal** [comment]
Review pending a green CI build. The current build hit the commit mismatch issue and will need to be re-triggered.

**Comment by mdharamadas1** [comment]
CI build in progress.

**Pull request merged**